tensorflow/python/ops/data_flow_ops.py

```diff
@@ -513,7 +513,8 @@ def close(self, cancel_pending_enqueues=False, name=None):
     operations will fail. Subsequent `dequeue` and `dequeue_many`
     operations will continue to succeed if sufficient elements remain
     in the queue. Subsequent `dequeue` and `dequeue_many` operations
-    that would block will fail immediately.
+    that won't block waiting for more elements to be enqueued and 
+    will fail immediately.
```

Comment:
Perhaps something like:

Subsequently `dequeue` and `dequeue_many` operations that would otherwise block waiting for more elements (if `close` hadn't been called) will now fail immediately.

will make it clearer.